mkdocs.yml

```diff
@@ -4,49 +4,63 @@
 #
 site_name: Deis Documentation
 pages:
-  - Welcome: index.md
-  - Quick Start: installing-workflow/quickstart.md
+  - Home: index.md
+  - Quick Start:
+    - Overview: quickstart/index.md
+    - Install CLI Tools: quickstart/install-cli-tools.md
+    - AWS:
+      - Boot: quickstart/provider/aws/boot.md
+      - DNS: quickstart/provider/aws/dns.md
+      - Install Workflow: quickstart/provider/aws/install-aws.md
+    - GKE:
+      - Boot: quickstart/provider/gke/boot.md
```

Comment:
We should change the title here. "Boot" seems like an odd choice of words as a title to install a GKE cluster (or AWS)